### PR TITLE
postfix: Add query filter for ldap aliases

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: modules/postfix/templates/virtual_alias_maps [hunk 1]
```diff
@@ -4,3 +4,6 @@ bind_dn = cn=write-user,dc=miraheze,dc=org
 bind_pw = <%= @ldap_password %>
 search_base = ou=people,dc=miraheze,dc=org
 scope = sub
+
+query_filter = (&(objectClass=postfixUser)(mail=%u))
+result_attribute = maildrop
```
Comment (JohnFLewis) [inline modules/postfix/templates/virtual_alias_maps:9]: Why are you using mail?

Comment (paladox) [reply modules/postfix/templates/virtual_alias_maps:9]: This didn't work, I was trying to prevent errors :)

Comment (JohnFLewis) [reply modules/postfix/templates/virtual_alias_maps:9]: That still doesn’t answer the question?

Comment (paladox) [reply modules/postfix/templates/virtual_alias_maps:9]: I was following an example I found and thought that could help.

Comment (JohnFLewis) [reply modules/postfix/templates/virtual_alias_maps:9]: But we don’t use mail for postfix at all, even if this did ‘work’ it would be reverted.

**Pull request closed without merging**